en/docs/webhooks.md

```diff
@@ -20,22 +26,14 @@ You can implement the logic and test it locally in the code server. After that y
 
 You can also use the Choreo built-in connectors in the webhook configuration.
```

Comment:
Is it correct to say "built-in connectors"? What we want to say is that it is possible to use any of the connectors that Choreo supports so far right?

Reply:
Changed this sentence.